projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-search.component.ts

```diff
@@ -31,6 +31,9 @@ export class IgxExcelStyleSearchComponent {
     @ViewChild('input', { read: IgxInputDirective })
     public searchInput: IgxInputDirective;
 
+    @Input()
+    public grid: any;
```

Comment:
I don't think you need the whole grid here. You only need displayDensity.
Also a good practice is to give specific type to properties, methods, parameters, etc. and don't use `any`. I suppose you've used `any` to avoid circular dependency but if you just pass displayDensity there won't be any.